src/firefly/js/tables/TableUtil.js

```diff
@@ -511,17 +534,7 @@ export function formatValue(col, val) {
     if (isNil(val)) return (nullString || '');
 
     if (Array.isArray(val)) {
-        const aryDim = (col.arraySize || '').split('x');
-
-        if (col.type === 'char' && aryDim.length > 0) {
-           aryDim.shift();    // remove first dimension because char array is presented as string
-        }
-
-        if (aryDim.length > 1) {
-            for(let i = 0; i < aryDim.length-1; i++) {
-                val = chunk(val, aryDim[i]);
-            }
-        }
+        val = convertToArraySize(col, val);
```

Comment:
@loitly Could you please review the changes to `TableUtil.js`? 

`convertToArraySize` converts one-dimensional as-stored array in an array column to multi-dimensional as-declared array.